Package.swift

```diff
@@ -216,6 +217,29 @@ let package = Package(
         .define("FIRAuth_MINOR_VERSION", to: "1.1"), // TODO: Fix version
       ]
     ),
+    .target(
+      name: "FirebaseCrashlytics",
+      dependencies: ["FirebaseCore", "FirebaseInstallations", "FBLPromises",
+                     "GoogleDataTransport", "nanopb"],
+      path: "Crashlytics",
+      sources: [
+        "Crashlytics/",
+        "Protogen/",
+        "Shared/",
+        "third_party/libunwind/dwarf.h",
+      ],
+      publicHeadersPath: "Crashlytics/Public",
+      cSettings: [
+        .headerSearchPath(".."),
+        .define("DISPLAY_VERSION", to: "0.0.1"), // TODO: Fix version
```

Comment:
Do we need to fix the version in this PR?

Reply:
Nope, version management is still TBD for Swift Package Manager. 

Will revisit further into the project.